src/e2e/java/teammates/e2e/cases/lnp/StudentProfileLNPTest.java

```diff
@@ -205,11 +208,11 @@ protected String getTestEndpoint() {
 
     @Override
     protected String getTestMethod() {
-        return "GET";
+        return HttpMethod.GET;
```

Comment:
You can abstract this to `BaseLNPTestCase`, like how it is done in `BaseActionTest`:

https://github.com/TEAMMATES/teammates/blob/7e3fa932a0b9dd7647e258a4ddd8c4b75bf14cf5/src/test/java/teammates/test/cases/webapi/BaseActionTest.java#L38-L44